crates/router/src/connector/utils.rs

```diff
@@ -1884,3 +1884,307 @@ mod error_code_error_message_tests {
         assert_eq!(error_code_error_message_none, None);
     }
 }
+
+#[macro_export]
+macro_rules! is_mandate_supported {
+    ($selected_pmd:expr, $pm_type:expr, $mandate_implemented_pmds:expr, $connector:expr) => {
+        if $mandate_implemented_pmds.contains(&PaymentMethodDataType::from($selected_pmd)) {
+            Ok(())
+        } else {
+            mandate_not_supported_error!($pm_type, $connector)
+        }
+    };
+}
```

Comment:
Can't this be a function?

Reply:
Yes, we can make it a function. Will do the necessary change.